targetconfig.json

```diff
@@ -717,6 +800,8 @@
         "Calliope Links": "calliope/links"
     },
     "teachertool": {
+        "showSharePageEvalButton": true,
```

Comment:
Set showSharePageEvalButton to false or remove it as it won't work with identity.

Reply:
I would also recommend removing code eval from the set of supported experiences: https://github.com/microsoft/pxt-calliope/blob/51034ac5ae7698ec55c8c81394e7d52d7cb9812d/pxtarget.json#L465

That way the user will get a clear error if they try to navigate to it manually.